mappings/net/minecraft/entity/ItemEntity.mapping

```diff
@@ -1,13 +1,14 @@
 CLASS net/minecraft/unmapped/C_uqpzijng net/minecraft/entity/ItemEntity
 	FIELD f_cejzztdx pickupDelay I
 	FIELD f_davsxlkm CANNOT_PICK_UP_DELAY I
-	FIELD f_laqmyrmj thrower Ljava/util/UUID;
+	FIELD f_laqmyrmj throwerUuid Ljava/util/UUID;
```

Comment:
I don't see the point of this change

Reply:
there's another `thrower` method that returns the entity